src/tests/ftest/launch.py

```diff
@@ -953,6 +958,161 @@ def rename_logs(avocado_logs_dir, test_file):
                 test_logs_dir, new_test_logs_dir, error))
 
 
+USE_DEBUGINFO_INSTALL = True
+
+def install_debuginfos():
+    """Install debuginfo packages"""
```

Comment:
Dotting the 'i's and crossing the 't's
```suggestion
    """Install debuginfo packages."""
```

Reply:
If I have to push for anything significant, I'll fix this up also.

Reply:
Yeah, not worth fixing unless a bigger change is needed.